### PR TITLE
feat/join-with-code

### PR DESCRIPTION
**Pull request by ngomesodent**

### CODE REVIEW

File: expo/app/(tabs)/rooms/index.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
-
 import React from "react";
 import { View } from "react-native";
+
 import Button from "../../../components/Button";
 
 export default function TabsRooms() {
+  // TODO: If the user is already in a room, the button "Rejoindre une salle" should redirect him to the room he is in.
```
Comment (MAXOUXAX) [inline expo/app/(tabs)/rooms/index.tsx:7]: While interesting, that hasn't been discussed before I think. Right now, nothing prevents a user from being in multiple active rooms at the same time.

File: expo/app/(tabs)/rooms/join.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    const { data: room, error: roomError } = await supabase
+      .from("active_rooms")
+      .select("id")
+      .eq("code", roomCode)
+      .single();
+
+    if (roomError) {
+      Keyboard.dismiss();
```
Comment (MAXOUXAX) [inline expo/app/(tabs)/rooms/join.tsx:30]: The keyboard should be dismissed when the user submits the form. Period.
It shouldn't be dismissed only when an error occurs

**Review by MAXOUXAX — CHANGES_REQUESTED** [review]
Just some tiny adjustements!

**Review by GaspardBBY — APPROVED** [review]
LGTM

**Review by TheRealSharKzy — COMMENTED** [review]
LGTM

**Review by TheRealSharKzy — APPROVED** [review]
LGTM

**Review by MAXOUXAX — APPROVED** [review]
Just a single comment to remove (because it's undiscussed) but otherwise it LGTM!

**Review by TheRealSharKzy — APPROVED** [review]
LGTM

**Review by MAXOUXAX — APPROVED** [review]
LGTM!!!

**Review by GaspardBBY — APPROVED** [review]
LGTM

**Pull request merged**